scripts/run-openapi-demo.sh

```diff
@@ -26,7 +26,9 @@ set -o allexport
 [[ -f $SIDETREE_PATH/.env ]] && source $SIDETREE_PATH/.env
 set +o allexport
 
+cd $AGENT_COMPOSE_FILE
+docker-compose -f docker-compose.yml  ${DEMO_COMPOSE_OP} -d
+cd $SIDETREE_COMPOSE_FILE
+docker-compose -f docker-compose.yml ${DEMO_COMPOSE_OP} -d
 cd $DEMO_PATH
-echo docker-compose -f docker-compose-demo.yml -f ${AGENT_COMPOSE_FILE} -f ${SIDETREE_COMPOSE_FILE} ${DEMO_COMPOSE_OP}
-docker-compose -f docker-compose-demo.yml -f ${AGENT_COMPOSE_FILE} -f ${SIDETREE_COMPOSE_FILE} ${DEMO_COMPOSE_OP}
-
+docker-compose -f docker-compose.yml ${DEMO_COMPOSE_OP} -d
```

Comment:
when do these fixtures get stopped?

Reply:
we have make clean and make clean-fixtures targets that will stop containers

Reply:
Why do we need to start with `-d` anyways?

We haven't previously done this.